database/youtube/Kubernetes.json

```diff
@@ -7,4 +7,4 @@
         "subcategory": "Kubernetes",
         "language": "english"
```

Comment:
This isn't needed for English sources

Reply:
Ig this is good to go @CBID2 . Thanks for the help!